Refactoring of AuditEventEntities

CTI777 opened the pull request:
Related to changes made in the issue:
https://github.com/Activiti/Activiti/issues/2636

CTI777 committed:
Refactoring of BPMNActivityAuditEventEntity

CTI777 committed:
Refactoring of ActivityCancelledAuditEventEntity

CTI777 committed:
Refactoring of ActivityCompletedAuditEventEntity

CTI777 committed:
Rerfatoring of ActivityStartedAuditEventEntity

CTI777 committed:
Refactoring of a test for ActivityStartedAuditEventEntity

CTI777 committed:
Refactoring of ProcessAuditEntities

CTI777 committed:
Refactoring of BPMNActivityAuditEventEntity

CTI777 committed:
Refactoring of SequenceFlowAuditEventEntity

CTI777 committed:
Refactoring of SignalReceivedAuditEventEntity

CTI777 committed:
Refactoring of VariableEventEntity

CTI777 committed:
Refactoring for TaskEventEntities

CTI777 committed:
Refactoring of TaskCandidateGroupEventEntities

CTI777 committed:
Refactoring of TaskCandidateUserEventEntities

CTI777 committed:
Refactoring of IntegrationEventEntity

Not clear exactly where is it used...

CTI777 committed:
Constructor removed

CTI777 committed:
Refactoring  of event converters

erdemedeiros commented on activiti-cloud-services-audit-jpa/src/main/java/org/activiti/cloud/services/audit/jpa/events/IntegrationEventEntity.java:
```diff
@@ -28,22 +28,14 @@
-        super(eventId,
-              timestamp,
-              eventType);
+        this.eventId = eventId;
```
Here it's better to keep using super (and making these three fields `private` instead of protected). I'll update it.

erdemedeiros committed:
Clean up

remove method addProcessContextInfoToEntityEvent from EventContextInfoAppender as everything is already set by the constructor
